includes/fraud-prevention/class-buyer-fingerprinting-service.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Class Buyer_Fingerprinting_Service
+ *
+ * @package WCPay\Fraud_Prevention
+ */
+
+namespace WCPay\Fraud_Prevention;
+
+/**
+ * Class Buyer_Fingerprinting_Service
+ */
+class Buyer_Fingerprinting_Service {
```

Comment:
Oh boy, this looks slim and nice. Can we also add a constructor, where we inject `Fraud_Prevention_Service` and turn all methods into non-static, please? On top, I'd suggest using a singleton pattern as `Fraud_Prevention_Service` does.

Reply:
And we need tests for the class.

Reply:
The `is_fraud_prevention_enabled_for_store` function was only a wrapper around the `Fraud_Prevention_Service::get_instance()->is_enabled()` so I believe we can use the later directly instead.

When using it, the class now is independent and doesn't need a `Fraud_Prevention_Service` part of the constructor. I transformed it to a singleton, but I think it's only a group of helper functions and we don't need it to be a singleton?

Reply:
After a talk on Slack p1649330319655879-slack-C02C00HCJ8J, I think we can keep it :)